build.gradle

```diff
@@ -93,6 +96,9 @@ def commonSpec = project.copySpec {
 	from(project.buildDir) {
 		include 'tmp/version.txt'
 	}
+	from("${buildDir}/docs/manpage") {
```

Comment:
I would do the inclusion in a separate PR as it also needs enablement in the various install scripts as otherwise its just "unnecessary bagage"

Reply:
ok